docs/wiki/99.-Developer.md

```diff
@@ -1,378 +1,394 @@
-clean                          clean the repository
-fix                            run autofixers
-install                        install library
-lint                           run lints
-test                           run the tests
-```
+Once you've built `csp` from a `git` clone, you will also need to
+configure `git` and your GitHub account for `csp` development.
 
+#### Configuring Git and GitHub for Development
 
-# Prerequisites
-`csp` has a few system-level dependencies which you can install from your machine package manager. Other package managers like `conda`, `nix`, etc, should also work fine. Currently, `csp` relies on the `GNU` compiler toolchain only.
+##### Create your fork
 
-# Building with Conda on Linux
+The first step is to create a personal fork of `csp`. To do so, click
+the "fork" button at https://github.com/Point72/csp, or just navigate
+[here](https://github.com/Point72/csp/fork) in your browser. Set the
+owner of the repository to your personal GitHub account if it is not
+already set that way and click "Create fork".
 
-The easiest way to get started on a Linux machine is by installing the necessary dependencies in a self-contained conda environment.
+##### Configure remotes
 
-Tweak this script to create a conda environment, install the build dependencies, build, and install a development version of `csp` into the environment.
+Next, you should set some names for the `git` remotes corresponding to
+main Point72 repository and your fork. If you started with a clone of
+the main `Point72` repository, you could do something like:
 
-## Install conda
 ```bash
-mkdir ~/github
-cd ~/github
-
-# this downloads a Linux x86_64 build, change your architecture to match your development machine
-# see https://conda-forge.org/miniforge/ for alternate download links
-
-wget https://github.com/conda-forge/miniforge/releases/download/23.3.1-1/Mambaforge-23.3.1-1-Linux-x86_64.sh
-chmod 755 Mambaforge-23.3.1-1-Linux-x86_64.sh
-./Mambaforge-23.3.1-1-Linux-x86_64.sh -b -f -u -p csp_venv
-
-. ~/github/csp_venv/etc/profile.d/conda.sh
+$ cd csp
+$ git remote rename origin upstream
+# for SSH authentication
+$ git remote add origin git@github.com:<username>/csp.git
+# for HTTP authentication
+$ git remote add origin https://github.com/ngoldbaum/csp.git
+```
 
-# optionally, run this if you want to set up conda in your .bashrc
-# conda init bash
+##### Authenticating with GitHub
 
-conda config --add channels conda-forge
-conda config --set channel_priority strict
-conda activate base
-```
+If you have not already configured `ssh` access to GitHub, you can find
+instructions to do so
+[here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh),
+including instructions to create an SSH key if you have not done
+so. Authenticating with SSH is usually the easiest route. If you are
+working in an environment that does not allow SSH connections to GitHub,
+you can look into [configuring a hardware
```

Comment:
PAT works too `https://<username>:<PAT>@github.com/...` I would just link to github docs

Reply:
Thanks, TIL!